src/OmniSharp/Models/NavigateResponse.cs

```diff
@@ -0,0 +1,9 @@
+namespace OmniSharp.Models
+{
+    public class NavigateResponse
+    {
+        public string FileName { get; set; }
```

Comment:
This will not end up in a different file. Does not try follow a partial, since there is no real order for the partial. Is there any use case you had in mind?